server/src/main/java/org/candlepin/resource/RootResource.java

```diff
@@ -71,7 +71,6 @@
         addResource(PoolResource.class);
         addResource(ProductResource.class);
         addResource(RulesResource.class);
-        addResource(StatisticResource.class, "statistics/generate");
```

Comment:
You can delete the `addResource` method that takes an object and string as well, since it looks like this resource was the only one using it.